src/frontend/js/utils/test/wrappers/BaseJoanieAppWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import { PropsWithChildren } from 'react';
+
+import JoanieSessionProvider from 'contexts/SessionContext/JoanieSessionProvider';
+import { IntlWrapper } from './IntlWrapper';
+import { ReactQueryWrapper } from './ReactQueryWrapper';
+import { AppWrapperProps } from './types';
+
+export const BaseJoanieAppWrapper = ({
```

Comment:
Let's call it `JoanieAppWrapper` WDYT ?

In this way we will be homogeneous over the code base. 
About session provider we have two versions : 
- `BaseSessionProvider`
- `JoanieSessionProvider`